src/it/scala/io/iohk/ethereum/rlp/RLPSpeedSuite.scala

```diff
@@ -11,18 +14,16 @@ import org.scalatest.FunSuite
 import org.scalatest.prop.{GeneratorDrivenPropertyChecks, PropertyChecks}
 import org.spongycastle.util.encoders.Hex
 
-import scala.language.implicitConversions
-
 /**
-  * Tests based on
-  *   - https://github.com/cryptape/ruby-rlp/blob/master/test/speed.rb
-  *   - https://github.com/ethereum/pyrlp/blob/develop/tests/speed.py
-  */
+ * Tests based on
+ *   - https://github.com/cryptape/ruby-rlp/blob/master/test/speed.rb
+ *   - https://github.com/ethereum/pyrlp/blob/develop/tests/speed.py
+ */
```

Comment:
I'm not trying to :fire: but the previous formatting style is the one that IntelliJ Idea creates, lol

Besides my meaningless comment, formatting commits might lead to difficult to read PRs. So, if we are doing to make a lot of this, I suggest to keep them in separate PRs.